apps/stack/template.yaml

```diff
@@ -378,6 +398,22 @@ Resources:
           - UseStackName
           - !Sub "${AWS::StackName}-MetricStream"
           - !Sub "${NameOverride}-MetricStream"
+  MetricsPollerRole:
+    Type: AWS::Serverless::Application
+    Condition: EnableMetricsPollerRole
+    Properties:
+      Location: ../externalrole/template.yaml
+      NotificationARNs:
+        - !Ref Topic
+      Parameters:
+        ObserveAwsAccountId: !Ref ObserveAwsAccountId
+        AllowedActions: >-
+          cloudwatch:GetMetricData,cloudwatch:ListMetrics,tag:GetResources
```

Comment:
You might want to parameterize `AllowedActions` and either default to this or leave it up to caller to define what actions are allowed.

Reply:
I like the idea of making it a default - I'll do that!